ifcbdb/dashboard/management/commands/updatetriggers.py

```diff
@@ -1,22 +1,73 @@
-from dashboard.models import Bin, bin_query
-from django.core.management.base import BaseCommand, CommandError
-from django.db import transaction
+from collections import deque
 import csv
 import os
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from tqdm import tqdm
+from tqdm._utils import _term_move_up
+
+from dashboard.models import Bin, bin_query
 
 class Command(BaseCommand):
 
-    help = 'change labels'
+    help = 'update triggers'
+
+    border = "="*50
+    clear_border = _term_move_up() + "\r" + " "*len(border) + "\r"
+
     def add_arguments(self, parser):
         parser.add_argument('-i','--input', type=str, help='Path to csv of mapping of of bin ID and trigger count')
-        parser.add_argument('-d', '--dataset', type=str, help='Name of dataset to process')
+        parser.add_argument('-d', '--dataset', type=str, help='Name of dataset to process(Optional)')
+
+    def last_line(self, filename):
+        d = deque(open(filename), 1)
+        try:
+            return d.pop()
+        except IndexError:
+            return None
+
+
+    def n_triggers(self, line):
+        if line is None:
+            return 0
+        return re.match(r'^(\d+)', line).group(0)
+
+    def bulk_update(self, chunk, pbar):
+        objs = []
+        for bin in chunk:
+            if not os.path.exists(bin.adc_path()):
+                pbar.write(self.clear_border + ("Error: Bins, " + str(objs) + " not updated! Continuing ..."))
```

Comment:
<img width="1428" alt="Screen Shot 2023-03-31 at 4 13 14 PM" src="https://user-images.githubusercontent.com/91488778/229220709-07a9d181-778e-417e-b2c6-b16212208bf8.png">
This hack found online, seems to push log messages above progress bar as shown in above image. (But the command run above the logs moves slightly). Let me know your thoughts on this.